src/DssSpell.sol

```diff
@@ -64,113 +109,217 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
 
-    // Operator address
-    address internal constant RWA015_A_OPERATOR          = 0x23a10f09Fac6CCDbfb6d9f0215C795F9591D7476;
-    // Custody address
-    address internal constant RWA015_A_CUSTODY           = 0x65729807485F6f7695AF863d97D62140B7d69d83;
+    uint256 internal constant THREE_PT_ONE_NINE_PCT_RATE  = 1000000000995743377573746041;
+    uint256 internal constant THREE_PT_FOUR_FOUR_PCT_RATE = 1000000001072474267302354182;
+    uint256 internal constant THREE_PT_NINE_FOUR_PCT_RATE = 1000000001225381266358479708;
+    uint256 internal constant FIVE_PT_FOUR_FOUR_PCT_RATE  = 1000000001679727448331902751;
+    uint256 internal constant FIVE_PT_SIX_NINE_PCT_RATE   = 1000000001754822903403114680;
+    uint256 internal constant SIX_PT_ONE_NINE_PCT_RATE    = 1000000001904482384730282575;
 
-    address internal constant RWA015_A_OUTPUT_CONDUIT    = 0xe3B3EAB84039D486b06980aed70607d394DF3827;
+    function actions() public override {
+        // ----- Deploy Multiswap Conduit for RWA015-A -----
 
+        // Add OPERATOR permission on RWA015_A_OUTPUT_CONDUIT
+        RwaOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).hope(RWA015_A_OPERATOR);
```

Comment:
Noting that we are only giving auth to the operator and not using the typical "Pause Proxy as fallback" pattern, and this new type of conduit _does not have `psm`_ filed via `hook` currently, nor is it done in this spell.

I guess that as long as pause proxy is authed on this contract, we could always `hope` and `nope` ourselves in the same spell, but I just wanted to raise this as this new output conduit format differs in that the `psm` value before the spell is cast is not a valid PSM value (it is the zero address).

Reply:
Tagging @0xdecr1pto to give an explainer here 🙂

Reply:
1. Pause proxy has admin permission(ward)
2. We don’t need to hook PSM as it’s suppose to be done by Operator